src/System.Private.Xml/src/System.Private.Xml.csproj

```diff
@@ -817,13 +821,13 @@
     <Compile Include="Workarounds\DummyTypes\PermissionSet.cs" />
     <Compile Include="Workarounds\DummyTypes\RequestCachePolicy.cs" />
     <Compile Include="Workarounds\DummyTypes\SecurityManger.cs" />
-    <Compile Include="Workarounds\SerializationEntry.cs" />
-    <Compile Include="Workarounds\ICloneable.cs" />
+    <None Include="Workarounds\SerializationEntry.cs" />
```

Comment:
>Include [](start = 10, length = 7)

if we don't need these workarounds, we should delete the source files entirely shouldn't we?